docs/reference/analysis/anatomy.asciidoc

```diff
@@ -1,5 +1,5 @@
 [[analyzer-anatomy]]
-== Anatomy of an analyzer
+=== Anatomy of an analyzer
```

Comment:
No substantive changes were made to this page. Simply adjusted headings and attributes so they would display normally.